homeassistant/components/ezviz/config_flow.py

```diff
@@ -369,7 +375,7 @@ async def async_step_reauth_confirm(
 
         data_schema = vol.Schema(
             {
-                vol.Required(CONF_USERNAME, default=entry.title): vol.In([entry.title]),
+                vol.Required(CONF_USERNAME, default=entry.unique_id): str,
```

Comment:
The unique_id is set above also to the serial number of the device.
I assume that the value of the serial attribute is not the username. Therefore, change the default value to the username, which is also stored in the config entry.

Reply:
Hi,

The reauth step is only for the EZVIZ cloud account which uses the username. Are you referring to the title placeholder variable?

https://github.com/home-assistant/core/blob/2cae79415dcd086c3ad1ed4d9ebf8b8b2cd00b1d/homeassistant/components/ezviz/strings.json#L3

Reply:
Think I understand what you mean, updated to the title_placeholder to use item.unique_id instead of entry.title.

Also updates entity.unique_id should the user change the username during reauth.

Tested changing the title of the cloud account entity and it works without issues.

Reply:
Are the serial number and the username equal?

Reply:
@RenierM26 I agree with @edenhaus this change is strange. Can you please explain what's happening so we can finish this PR?

#### Without this PR:
- We ask the user to provide a "username", but the schema will fail unless the user enters the title of the config entry. This makes little sense.

#### With this PR:
- We ask the user to provide a "username", which defaults to the serial number of the device(?). Why do we ask the user to enter the serial number in a labelled with `Username`?